recipes/donfig/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "donfig" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 669f3b6f88536c8b8f7aae459d59f249c7a7a4457c8288dded12d4fa61d935fa
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pyyaml
+  run:
+    - python
+    - pyyaml
+
+test:
+  imports:
+    - donfig
+    - donfig.tests
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/pytroll/donfig
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Python package for configuring a python package
+  doc_url: https://donfig.readthedocs.io/en/latest/
+  dev_url: https://github.com/pytroll/donfig
+
+extra:
+  recipe-maintainers:
+    - djhoese
+    - mraspaud
+    - pnuu
+    - adybbroe
+    - mrocklin
+    - jcrist
+    - jhamman
```

Comment:
@jhamman @adybbroe @pnuu @mraspaud - please let us know if you are OK being maintainers

Reply:
fine by me.